app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarScreen.kt

```diff
@@ -422,21 +428,46 @@ fun FilterBox(
     }
 }
 
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun Week(
     modifier: Modifier = Modifier,
     tasks: List<Task>
 ) {
-    Row(
-        modifier = modifier
-            .horizontalScroll(rememberScrollState())
-            .fillMaxWidth()
-    ) {
-        (1..7).forEach {
-            Day(
-                name = DayOfWeek.of(it).name,
-                tasks = tasks.filter { task -> task.scheduledAt.dayOfWeek.value == it }
-            )
+    val days = (1..7).map { DayOfWeek.of(it).name }
+    val pagerState = rememberPagerState(pageCount = { days.size })
+    val currentPage = pagerState.currentPage
+
+    @Composable
+    fun DayPage(day: String, pageIndex: Int) {
+        val tasksForDay = tasks.filter { task -> task.scheduledAt.dayOfWeek.name == day }
+        val scale = animateFloatAsState(
+            targetValue = if (pageIndex == currentPage) 1f else 0.8f,
+            animationSpec = spring(stiffness = 500f)
+        ).value
+
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .graphicsLayer(scaleX = scale, scaleY = scale),
+            contentAlignment = Alignment.Center
+        ) {
+            Day(name = day, tasks = tasksForDay)
         }
     }
-}
\ No newline at end of file
+
+
+    @Composable
+    fun Page(pageIndex: Int) {
+        val day = days[pageIndex]
+        DayPage(day, pageIndex)
```

Comment:
Single line required between variable initialization and component call.

Also, could just be removed if all it does is just wrap another component. Then, in HorizontalPager iteration just call like this:
```kotlin
DayPage(days[page], page)
```